packaging/rpm/cri-resource-manager.spec.in

```diff
@@ -28,14 +31,15 @@ make build BUILD_DIRS=cri-resmgr
 %make_install UNITDIR=%{_unitdir} SYSCONFDIR=%{_sysconfdir} BUILD_DIRS=cri-resmgr
 install -m 0700 -d %{?buildroot}%{_sharedstatedir}/cri-resmgr
 
-
 %files
 %defattr(-,root,root,-)
 %{_bindir}/*
 %{_sysconfdir}/sysconfig/*
 %{_unitdir}/*
 %dir %attr(0700,root,root) %{_sharedstatedir}/cri-resmgr
+%dir %attr(0700,root,root) %{_sysconfdir}/cri-resmgr
+%{_sysconfdir}/cri-resmgr/*.sample
```

Comment:
well, sample is now in %doc which is good. but here we still have same sample.
in theory, as we ship with default parameter that refer to fallback.cfg 
( `CONFIG_OPTIONS="--fallback-config /etc/cri-resmgr/fallback.cfg"` ), we need to ship here fallback.cfg, without sample suffix.

Reply:
Well, that's intentional. I don't want the user to be able to start cri-resmgr after installation without having to even touch the configuration file. So the idea here is that we merely put a sample/template for the configuration in the right directory which the user at least needs to copy in place (hopefully also taking a look at it) the first time cri-resmgr is installed, before it can be successfully started.